src/block/block_compact.c

```diff
@@ -223,6 +226,12 @@ __block_compact_estimate_remaining_work(WT_SESSION_IMPL *session, WT_BLOCK *bloc
     /* We should estimate the work only once. */
     WT_ASSERT(session, !block->compact_estimated);
 
+    /* Output the estimation logs as info if we're in dryrun mode. */
```

Comment:
We should check the current level, only change it if it's less than `WT_VERBOSE_DEBUG_2` and revert it back to the original value if required.

Reply:
You can use `WT_VERBOSE_SET_AND_SAVE` and `WT_VERBOSE_RESTORE` for this.

Reply:
I'm not sure if I understand this, why do we want to change the session verbosity level? The changes make the level of the log messages themselves change. Not the log level of the components at the session level.

Reply:
I see, my bad, please ignore!